### DISCUSSION TITLE
Is it possible to dynamically build MathTex() with provided args?

### DISCUSSION THREAD

**Discussion by Rassska** [3b1b/manim#1768] [Q&A] [answered]
Hi, everyone!
I'm just wondering, can I dynamically create MathTex objects, let's say in the following way:

![Screenshot from 2022-03-28 22-19-17](https://user-images.githubusercontent.com/73281386/160470844-4cfbd633-7c2e-40ca-b5a0-47136ef04c6e.png)

For those, who can't open the image: 
`for i in range(1, 3):`
 `for j in range(0, 3):`
 `dpMax = MathTex(r"dp[$i][$j] = \max(matrix[$i][$j] + dp[$(i - 1)][$j])").set_color_by_gradient("#33ccff","#ff00ff")`

By now, I only have this dirty solution for that:
![Screenshot from 2022-03-28 22-31-03](https://user-images.githubusercontent.com/73281386/160472670-485331af-a612-44b8-9f72-eba26167af40.png)

**Comment by TonyCrane** [comment, marked as answer]
I think you just need a string format syntax in python, it has nothing to do with manim.

> `f"a{1+1}b"` is the same as `"a2b"`, try it.

**Reply by Rassska** [reply]
I got it, thanks a lot, it's hard to distinguish after kotlin..
